chef_master/source/install_server_ha.rst

```diff
@@ -492,22 +520,49 @@ See https://www.elastic.co/guide/en/elasticsearch/reference/current/setup-config
-* ``leaderl.runsv_group``  The group that sensitive password files will belong to.  This is used internally for test purposes and should never be modified otherwise.  ``'chef_pgsql'`` by default.
-* ``leaderl.status_internal_update_interval_seconds``  How often we check for a change in the leader service's status.  5 seconds by default.
-* ``leaderl.status_post_update_interval_seconds`` How often etcd is updated with the leader service's current status.  10 seconds by default.
-* ``leaderl.username`` ``'chef_pgsql'``
-* ``leaderl.log_rotation.max_messages_per_second``  Rate limit for the number of messanges that the Erlang error_logger will output.  ``1000`` by default.
-* ``leaderl.etcd_pool.ibrowse_options``  Internal options to affect how requests to etcd are made (see https://github.com/cmullaparthi/ibrowse/blob/master/doc/ibrowse.html).
+* ``leaderl.leader_ttl_seconds``  The number of seconds it takes the
+  leader key to expire.  Increasing this value will increase the
+  amount of time the cluster will take to recognize a failed leader.
+  Lowering this value may leade to frequent leadership changes and
+  thrashing. ``30`` by default (``10`` by default before 1.2).
+* ``leaderl.required_active_followers`` The number of followers that
+  must be syncing via a PostgreSQL replication slot before a new
+  leader will return 200 to /leader HTTP requests.  If an existing
+  leader fails to maintain this quorum of followers, the /leader
+  endpoint will return 503 but active connections will still be able
+  to complete their writes to the database.  0 by default.
+* ``leaderl.runsv_group`` The group that sensitive password files will
+  belong to.  This is used internally for test purposes and should
+  never be modified otherwise.  ``'chef_pgsql'`` by default.
+* ``leaderl.status_internal_update_interval_seconds`` How often we
+  check for a change in the leader service's status.  5 seconds by
+  default.
+* ``leaderl.status_post_update_interval_seconds`` How often etcd is
+  updated with the leader service's current status.  10 seconds by
+  default.
+* ``leaderl.username 'chef_pgsql'``
+* ``leaderl.log_rotation.max_messages_per_second`` Rate limit for the
+  number of messanges that the Erlang error_logger will output.
+  ``1000`` by default.
+* ``leaderl.etcd_pool.ibrowse_options`` Internal options to affect how
+  requests to etcd are made (see
+  https://github.com/cmullaparthi/ibrowse/blob/master/doc/ibrowse.html).
 
 Chef HA backend leader health status settings
 -----------------------------------------------------
-* ``leaderl.health_check.interval_seconds``  How frequently, in seconds, to poll the service for health status.  2 by default.
+* ``leaderl.health_check.interval_seconds`` How frequently, in
+  seconds, to poll the service for health status.  We recommend
+  setting this to at least 5 times the value of
```

Comment:
We recommend setting leader_ttl_seconds to a value at least 5 times larger than interval_seconds - i.e. you want followers to give the leader 5 intervals worth of time to be temporarily unresponsive before triggering a fail-over.